test/Dapr.Client.Test/SecretApiTest.cs

```diff
@@ -11,6 +11,8 @@
 // limitations under the License.
 // ------------------------------------------------------------------------
 
+using FluentAssertions.Equivalency;
```

Comment:
I see this using is outside of the namespace, where the `using System;` is within. How much do we care about consistency?

Reply:
This isn't in use by anything, so I've just removed it in the latest commit. Undoubtedly an over-eager addition by the IDE.